src/rpc/server.cpp

```diff
@@ -11,14 +11,18 @@
 #include <util/strencodings.h>
 #include <util/string.h>
 #include <util/system.h>
+#include <util/time.h>
 
 #include <boost/signals2/signal.hpp>
 
 #include <cassert>
-#include <memory> // for unique_ptr
+#include <chrono>
+#include <memory>
 #include <mutex>
 #include <unordered_map>
 
+using SysClock = std::chrono::system_clock;
```

Comment:
That's not steady_clock, right?

Reply:
Looks like a data point to show my brain stops working at 31°C

Reply:
Heeh same over here. Ugh.